pylutron/__init__.py

```diff
@@ -381,9 +380,10 @@ def _parse_cci(self, keypad, component_xml):
     cci_xml = component_xml.find('CCI')
     cci_type = cci_xml.get('ButtonType')
     led_logic = cci_xml.get('LedLogic')
+    name = f"keypad {keypad.id}: CCI {component_number}"
```

Comment:
see above.

Reply:
see above

Reply:
modified. HA has to compose the unique name